package proto;

message Cpfa {
	required double probabilityOfSwitchingToSearching = 1;
	required double probabilityOfReturningToNest = 2;
	required double uninformedSearchVariation = 3;
	required double rateOfInformedSearchDecay = 4;
	required double rateOfSiteFidelity = 5;
	required double rateOfLayingPheromone = 6;
	required double rateOfPheromoneDecay = 7;
}
